standalone-metastore/metastore-common/src/main/java/org/apache/hadoop/hive/metastore/conf/MetastoreConf.java

```diff
@@ -1825,6 +1825,36 @@ public enum ConfVars {
         "hive.metastore.properties.servlet.auth", "jwt",
         "Property-maps servlet authentication method (simple or jwt)."
     ),
+    ICEBERG_CATALOG_SERVLET_PATH("hive.metastore.catalog.servlet.path",
```

Comment:
iceberg's RestCatalog API is versioned. 
https://github.com/apache/iceberg/blob/main/open-api/rest-catalog-open-api.yaml
How do you plan to support that?